### ISSUE TITLE
Missing documentation

### ISSUE THREAD

**Issue by Dr-Terrible** [cardoe/cargo-ebuild#3]
Please, can you add some kind of documentation that explains how to use and invoke the command-line tool for a real case scenario? People not used to rust/cargo will have hard times figuring it out. Thank you.

**Comment by cardoe** [comment]
Sure. Basically right now its git clone the project and checkout the tag of the version you want to build and run `cargo ebuild`. Obviously those steps are a bit much and it'd be nicer to run `cargo ebuild <pkg>` and have it reach out to crates.io. That's something I need to add.

**Comment by Dr-Terrible** [comment]
> Obviously those steps are a bit much and it'd be nicer to run cargo ebuild <pkg> and have it reach out to crates.io.

Indeed, that was my initial confusion. I spent at least an hour trying to figure out why the tool complained and did nothing when I invoked `cargo ebuild <github-url>`. Only later I realised that I had to clone the repo and run the command on the clone.

> That's something I need to add.

That would be awesome, thank you. It's not urgent, btw; I'm fine with manually checking out a repo.
